README.md

```diff
@@ -1,372 +1,742 @@
-            767.67676768,  777.77777778,  787.87878788,  797.97979798,
-            808.08080808,  818.18181818,  828.28282828,  838.38383838,
-            848.48484848,  858.58585859,  868.68686869,  878.78787879,
-            888.88888889,  898.98989899,  909.09090909,  919.19191919,
-            929.29292929,  939.39393939,  949.49494949,  959.5959596 ,
-            969.6969697 ,  979.7979798 ,  989.8989899 , 1000.        ])
+    array([   0.        ,   41.66666667,   83.33333333,  125.        ,
+            166.66666667,  208.33333333,  250.        ,  291.66666667,
+            333.33333333,  375.        ,  416.66666667,  458.33333333,
+            500.        ,  541.66666667,  583.33333333,  625.        ,
+            666.66666667,  708.33333333,  750.        ,  791.66666667,
+            833.33333333,  875.        ,  916.66666667,  958.33333333,
+           1000.        ])
 
 
 
-Just as expected. 100 equally spaced numbers starting from 0 to a 1000. 
+As you can see, this code produced 25 equally spaced numbers from 0 to 1000.
 
-After preparing the the data, we can use matplotlib's `plot()` function to create the plot with our data, `legend()` to add context information to the plot, and finally `show()` functions to output the plot . 
+## `matplotlib` Line Plot
 
-In jupyter notebooks, you can use `%matplotlib` magic with `inline` to show plots inside the notebook or `qt` for external/interactive plots. `inline` is recommended for most needs. There is also a `% matplotlib notebook` magic, which we'll see shortly.
+Throughout this lesson we'll be using a new type of plot in our examples: a *line plot*.
 
+### What Are Line Plots?
 
-```python
-# Set plot space as inline for inline plots and qt for external plots
-%matplotlib inline
+A line plot (or line graph) is a two-dimensional data visualization that is used to represent a trend. The x-axis should represent something that "increases" — either a time measure like seconds, hours, or years or an ordinal measure like finishing places in a competition (1st, 2nd, 3rd, etc.).
+
+Line plots look kind of like scatter plots, except that each point is connected by a trend line.
+
+### Line Plots in Matplotlib
 
-# Use plot() function to create a plot using above values on both x and y coordinates. Add a label.
-plt.plot(data, data, label='Sample Data')
+The function to make a line plot in `matplotlib` is just `.plot()` ([documentation here](https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.plot.html#matplotlib.axes.Axes.plot)). Let's plot our generated data and set a legend:
 
-# Add a legend to the plot with legend()
-plt.legend()
 
-# Output the final plot
-plt.show()
+```python
+# Create the plot
+fig, ax = plt.subplots()
+
+# Use plot() function to create a plot using above values
+ax.plot(years, data)
+
+# Add a legend to the plot
+ax.legend(["Sample Data"]);
 ```
 
 
-![png](index_files/index_4_0.png)
+    
+![png](index_files/index_7_0.png)
+    
 
 
```

Comment:
Added a note about the use of the semicolon in preventing the printing of the last graph object to be created. 

In the future, the print statement which the semicolon suppresses may not show up visually in every way this code might be hosted / distributed to students in the future (eg illumidesk, docker, etc), but I worded it so that even if it doesn't show up in whatever instance students encounter this code in the future, it should make sense: 

**Note**: Notice the semicolon at the end of the last line.  If this is not included, the location in memory of the last object to be created in the graph can be printed before the graph is displayed.  

If you re-run the above cell without the semicolon, you may see something like `<matplotlib.legend.Legend at 0x120f17910>` printed above the line graph. 

In order to suppress this print statement, just include a semicolon at the end of the last line of a given visualization!

Reply:
Added